pkgs/development/interpreters/starlark/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+buildGoModule rec {
+  pname = "starlark";
+  version = "unstable-2022-02-13";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "starlark-go";
+    rev = "c55a923347b1ca17cb609b1570187075c6bb911f";
+    sha256 = "sha256-Str7PKQqbI0SniRJDompwlshHmC299EZzqEb4pYsd+8=";
+  };
+
+  vendorSha256 = "sha256-lgL5o3MQfZekZ++BNESwV0LeoTxwEZfziQAe99zm4RY=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/google/starlark-go";
+    description = "Starlark in Go: the Starlark configuration language, implemented in Go";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ aaronjheng ];
```

```suggestion
    license = licenses.bsd3;
    platforms = platforms.all;
    maintainers = with maintainers; [ aaronjheng ];
```
Without that the package doesn't get prebuilt binaries.

---

The default for `buildGoModule` is `platforms.all`.

---

No, it is ``platforms.linux ++ platforms.darwin`` because go does not work currently on any other platform.